openedx/adg/lms/applications/models.py

```diff
@@ -442,3 +438,45 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class AdminNote(TimeStampedModel):
+    """
+    Model to save the notes of admin on the user application.
+    """
+    user_application = models.ForeignKey(
+        UserApplication, related_name='notes', on_delete=models.CASCADE, verbose_name=_('User Application'),
+    )
+    saved_by = models.ForeignKey(User, on_delete=models.CASCADE, verbose_name=_('Saved By'), )
+    note = models.TextField(verbose_name=_('Note'))
+
+    class Meta:
+        app_label = 'applications'
+        ordering = ['-created']
+
+    def __str__(self):
+        return f'{self.id} {self.note}'
+
+
+class ApplicantMessage(TimeStampedModel):
+    """
+    Model to save the messages of admin for the applicant on the user application.
+    """
+    user_application = models.ForeignKey(
+        UserApplication,
+        related_name='applicant_messages',
```

Comment:
ApplicantMessage suggests that this message was from applicant, maybe we can rename it to something like `MessageForApplicant` or `AdminMessage`. If we decide to change the name we should also change the related_name.

Reply:
I'd recommend the former, i.e. `MessageForApplicant`.